include/IECoreImage/ImageReader.h

```diff
@@ -118,6 +122,7 @@ class IECOREIMAGE_API ImageReader : public IECore::Reader
 
 		IECore::StringVectorParameterPtr m_channelNamesParameter;
 		IECore::BoolParameterPtr m_rawChannelsParameter;
+		IECore::IntParameterPtr m_miplevelParameter;
```

Comment:
Just noting that if we needed to, we could avoid the breaking change by looking the parameter up by name.